docs/common/envs.rst

```diff
@@ -0,0 +1,31 @@
+.. _envs:
+
+.. automodule:: stable_baselines3.common.envs
+
+
+
+Custom Environments
+===================
+
+Those environments were created for testing purposes.
+
+
+BitFlippingEnv
+--------------
+
+.. autoclass:: BitFlippingEnv
+  :members:
+
+
+SimpleMultiObsEnv
+-----------------
+
+.. autoclass:: SimpleMultiObsEnv
+  :members:
+
+
+NineRoomMultiObsEnv
```

Comment:
do we really need the two?

Reply:
Probably not. I'll take it out on the next push.